packages/integration-karma/scripts/karma-plugins/env.js

```diff
@@ -29,19 +29,19 @@ function createEnvFile() {
         fs.mkdirSync(DIST_DIR);
     }
 
-    const content = [
-        `window.process = {`,
-        `    env: {`,
-        `        NODE_ENV: "development",`,
-        `        COMPAT: ${COMPAT},`,
-        `        DISABLE_SYNTHETIC: ${DISABLE_SYNTHETIC},`,
-        `        FORCED_NATIVE_SHADOW_MODE_FOR_TEST: ${FORCED_NATIVE_SHADOW_MODE_FOR_TEST},`,
-        `        NATIVE_SHADOW_MODE: ${DISABLE_SYNTHETIC || FORCED_NATIVE_SHADOW_MODE_FOR_TEST},`,
-        `        NATIVE_SHADOW_ROOT_DEFINED: typeof ShadowRoot !== "undefined"`,
-        `    }`,
-        `};`,
-    ];
-    fs.writeFileSync(ENV_FILENAME, content.join('\n'));
+    fs.writeFileSync(
+        ENV_FILENAME,
+        `
+        window.process = {
+            env: {
+                NODE_ENV: 'development',
+                COMPAT: ${COMPAT},
+                NATIVE_SHADOW: ${!SYNTHETIC_SHADOW_ENABLED || FORCE_NATIVE_SHADOW_MODE_FOR_TEST},
```

Comment:
This is a computed env variable which I'm not excited about but I couldn't think of a better alternative.

Reply:
Actually, we can move the `COMPAT` and `NATIVE_SHADOW` values into a new `process.test` object to make it clear that it isn't an env variable. It was a bad idea earlier before we move to using a feature flag internally because it would force users to define it in order to run LWC (i.e., `process.test = {}`).

Thoughts?